engine/src/main/java/org/camunda/bpm/engine/impl/cmmn/entity/repository/CaseDefinitionEntity.java

```diff
@@ -46,11 +48,13 @@
 
   protected Map<String, TaskDefinition> taskDefinitions;
 
+  protected Map<String,TimerJobDeclaration<?>> timerJobDeclarationMap;
```

Comment:
We should not cache the job declaration at the definition level as well. So please remove it.